scripts/generate-rpi-imager-json.py

```
@@ -0,0 +1,406 @@
+    """Generate Raspberry Pi Imager JSON from Armbian images JSON"""
+
+    # Source URL for all Armbian images
+    ALL_IMAGES_URL = "https://github.armbian.com/armbian-images.json"
+
+    # Ubuntu releases
+    UBUNTU_RELEASES = {"jammy", "noble", "plucky"}
+
+    # Raspberry Pi boards to filter (board_slug values)
+    RPI_BOARDS = {
+        "rpi4b"
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)